packages/contracts/script/PostDeploy.s.sol

```diff
@@ -35,26 +34,18 @@ contract PostDeploy is Script {
     world.installRootModule(new StandardDelegationsModule(), new bytes(0));
     world.installModule(new PuppetModule(), new bytes(0));
 
-    // Register and mint ERC20 token
-    IERC20Mintable token = registerERC20(
-      world,
-      "BugToken",
-      ERC20MetadataData({ decimals: 18, name: "TEST_BUGS", symbol: "TEST_BUGS" })
-    );
-
-    // Transfer ownership of the token namespace to the world contract
-    ResourceId namespaceId = WorldResourceIdLib.encodeNamespace("BugToken");
-    world.transferOwnership(namespaceId, worldAddress);
+    // Register public materials
+    PublicMaterials.init();
```

Comment:
Currently this breaks the script because too much to initialize I guess (each material gets a new module)
It's solved by adding `--slow` to `forge script`, which is complicated currently, since mud's test, deploy etc don't allow passing script options
https://github.com/latticexyz/mud/pull/2703 should make it easier